tools/googlesearch/searchTool.go

```diff
@@ -0,0 +1,48 @@
+package googlesearch
```

Comment:
new packages should have doc.go files, please keep all filename lowercase, use underscores if you'd like.